manifests/crd.yaml

```diff
@@ -79,11 +79,11 @@ spec:
                 one / all / fixed / fixed-percent / random-max-percent'
               type: string
             nextRecover:
-              description: Next time when this action will be recovered
+              description: Next time when this action will be recovered.
```

Comment:
revert these changes?

Reply:
This is auto-generated by executing `make yaml`.